tffm/models.py

```diff
@@ -81,12 +82,16 @@ class TFFMRegressor(TFFMBaseModel):
     This class implements L2-regularized arbitrary order FM model with MSE
     loss and gradient-based optimization.
 
+    Custom loss functions are not supported, mean squared error is always
+    used. Any loss function provided in parameters will be overwritten.
+
     See TFFMBaseModel and TFFMCore docs for details about parameters.
     """
 
     def __init__(self, **init_params):
-        assert 'loss_function' not in init_params
+        # overwrite any custom loss function
```

Comment:
Logic with overwriting is not transparent and we should avoid it, I believe.

Reply:
I suppose the better solution would be to raise an error if `loss_function` or `class_weight` was in `init_params`. Similar to the last one, we can stick to assertions if you prefer, but I find them ambiguous.